src/internal.c

```diff
@@ -24411,10 +24409,37 @@ const char* wolfSSL_ERR_reason_error_string(unsigned long e)
     case HTTP_APPSTR_ERR:
         return "HTTP Application string error";
 #endif
-#ifdef OPENSSL_EXTRA
+#if defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL) || \
```

Comment:
The OPENSSL_EXTRA_X509_SMALL initial work was trimmed where every byte of code size mattered. Can this section of extra error messages not be included for OPENSSL_EXTRA_X509_SMALL?